packages/design-system/src/styles/_base.scss

```diff
@@ -156,73 +156,74 @@ $breakpoints: (
 
 @mixin heading--5xl {
   @include heading;
-  font-size: var(--font-size-3xl);
+  font-size: var(--typography-heading-5xl__font-size--mobile);
 
   @media (min-width: $media-width-sm) {
-    font-size: var(--font-size-4xl);
+    font-size: var(--typography-heading-5xl__font-size--tablet);
   }
 
   @media (min-width: $media-width-md) {
-    font-size: var(--font-size-5xl);
+    font-size: var(--typography-heading-5xl__font-size);
   }
 }
 
 @mixin heading--4xl {
   @include heading;
-  font-size: var(--font-size-3xl);
+  font-size: var(--typography-heading-4xl__font-size--mobile);
   font-weight: var(--typography-heading-4xl__font-weight);
 
   @media (min-width: $media-width-md) {
-    font-size: var(--font-size-4xl);
+    font-size: var(--typography-heading-4xl__font-size);
   }
 }
 
 @mixin heading--3xl {
   @include heading;
   @include heading-margins;
 
-  // There's no font size token for the mobile size heading--3xl style
-  // There's no space in our current font size token names to include it (what do you call a token between font-size-2xl and font-size-3xl?)
-  // Related to https://jira.cms.gov/browse/WNMGDS-2174
-  font-size: 1.875rem;
+  font-size: var(--typography-heading-3xl__font-size--mobile);
   font-weight: var(--typography-heading-3xl__font-weight);
 
   @media (min-width: $media-width-md) {
-    font-size: var(--font-size-3xl);
+    font-size: var(--typography-heading-3xl__font-size);
   }
 }
 
 @mixin heading--2xl {
   @include heading;
   @include heading-margins;
-  font-size: var(--font-size-2xl);
+  font-size: var(--typography-heading-2xl__font-size--mobile);
+
+  @media (min-width: $media-width-md) {
+    font-size: var(--typography-heading-2xl__font-size);
+  }
 }
 
 @mixin heading--xl {
   @include heading;
   @include heading-margins;
-  font-size: var(--font-size-xl);
+  font-size: var(--typography-heading-xl__font-size);
```

Comment:
🍕 and here we're saying the starting point is the un-suffixed font size. i guess what confuses me is 

1. by excluding a suffix on some tokens, are we making an assumption that token is the "base" or "default"?
2. are un-suffixed tokens the "default" size or the desktop size (or both)?
3. the combination of our new size tokens and media query sizes is a little confusing to me (mixing "--tablet" and `$media-width-sm`, "default" and `$media-width-md`)
    1. this might be a personal thing, but when i think of medium viewports, i think tablet
    2. i wonder if this will cause issues in the future, e.g., needing size X that happens to have a token name of --tablet as the starting point font size 

tl;dr the code gets hard to reason about when combining non-suffixed and viewport-suffixed tokens with media queries using a different sizing unit (sm, md, lg, etc)

Reply:
Yeah, and naming them can't be based on the screen size name like `md` because then we'd have `size-md--md`. Naming is hard. I'm trying to capture what the CSS looks like now in tokens without modifying the tokens we already have for `--font-size-*`. My hope is that at some point in the future we're getting rid of the `--mobile` and `--tablet` variables and using consistent sizing